datafusion/expr-common/src/signature.rs

```diff
@@ -135,6 +134,8 @@ pub enum TypeSignature {
     /// Null is considerd as `Utf8` by default
     /// Dictionary with string value type is also handled.
     String(usize),
+    /// Zero argument
+    NullAry,
```

Comment:
Is the `A` here intended to be upper-case? I'd have expected `Nullary`

Reply:
`Nullary` looks good to me too

Reply:
- I made a PR to fix: https://github.com/apache/datafusion/pull/13817